sdk/sdktypes/integration_id.go

```diff
@@ -23,7 +24,16 @@ func StrictParseIntegrationID(s string) (IntegrationID, error) { return Strict(P
 func IsIntegrationID(s string) bool { return IsIDOf[integrationIDTraits](s) }
 
 func IntegrationIDFromName(name string) IntegrationID {
-	txt := fmt.Sprintf("%s_%026x", integrationIDKind, kittehs.HashString64(name))
+	const chars = "0123456789abcdefghjkmnpqrstvwxyz"
```

Comment:
I have a couple of technical comments on how you implemented this, but more importantly this function doesn't guarantee uniqueness, so we can't use it in production.

Example: "httpi", "httpl", and "httpx" would all be converted to "httpx".

Reply:
yeah hence why it's only for builtin integrations, where it's not such an issue imo